### PR TITLE
Adding social media share links to footer

### LINKED ISSUE #136: ✨Social Media images
## Description
We should add some sort of images that should be used when it is shared on social media.

### PR DESCRIPTION
**Pull request by deadSnowman**
# Description
Added Facebook and Twitter social media links to footer
Added name and Github link in CONTRIBUTING.md

# Screenshots
Annotation 2019-10-02 160750

# Issues
Closes #136

### CODE REVIEW

**Comment by damcclean** [comment]
Hey, thanks for your contribution!

Although, I was looking for the meta sharing images instead of sharing buttons...

**Comment by deadSnowman** [comment]
image

Digital Ocean's click to tweet
https://ctt.ac/c5WeF

New one referencing Hacktoberfest finder
https://ctt.ac/andh5

I'm not quite sure why the digital ocean URL in the tweet was auto changed to a click to tweet link.  Let me know if you want to change the description or revert to the DigitalOcean click to tweet.  Same with the Facebook link.  It now references https://hacktoberfest-finder.netlify.com/ instead of the digital ocean page, which I thought was what you would prefer.

**Comment by deadSnowman** [comment]
> Although, I was looking for the meta sharing images instead of sharing buttons...

I'm not sure what you mean.  Were you looking for something to mimic the digital ocean site?
image

**Comment by damcclean** [comment]
No sorry.

When you go to share the webpage on places like Twitter or Facebook, it's missing an image.

For example, it's this image that we currently don't have:
image

File: index.html [hunk 1]
```diff
@@ -87,12 +89,18 @@ <h3 class="text-center text-blue mb-4 break-words">{{ result.user.login }}/{{ re
     <footer class="bg-primary w-full mt-8">
       <div class="container mx-auto flex flex-row items-center justify-between p-8">
         <p class="text-amber">Unofficial site by <a class="text-secondary" href="http://duncan.mcclean.co.uk">Duncan McClean</a> and <a class="text-secondary" href="https://github.com/damcclean/hacktoberfest-finder/blob/master/CONTRIBUTORS.md">contributors</a>.</p>
-
+        <div class="flex flex-row items-center justify-center">
+            <a class="mx-2" href="https://www.facebook.com/sharer/sharer.php?u=https://hacktoberfest-finder.netlify.com/">
```
Comment (damcclean) [inline index.html]: You can remove this now...

**Comment by damcclean** [comment]
Awesome! Thanks for your contribution.

**Pull request merged**